src/pages/post-feed/CreatPost.tsx

```diff
@@ -2,12 +2,12 @@
 import Image from 'next/image';
 
 export default function CreatPost() {
-  interface User {
+  interface UserList {
```

Comment:
this name is misleading, it is referring to a list, yet the object is singular, so a name User was a better name for it